Marlin/Marlin_main.cpp

```diff
@@ -2257,22 +2258,49 @@ static void clean_up_after_endstop_or_probe_move() {
       }
     #endif
 
-    // move down slowly to find bed
-    if (do_probe_move(-10, Z_PROBE_SPEED_SLOW)) return NAN;
-
-    #if ENABLED(DEBUG_LEVELING_FEATURE)
-      if (DEBUGGING(LEVELING)) DEBUG_POS("<<< run_z_probe", current_position);
+    #if MULTIPLE_PROBING > 2
+      float probes_total = 0;
+      for (uint8_t p = MULTIPLE_PROBING + 1; --p;) {
     #endif
```

Comment:
Is this a stray endif?

Reply:
I'd say no... Considering the line numbers, theyreplaced 2264 with 2261>2263 and the new 2264 is the old 2265 so it's the end of the first if that got replaced with the new one